src/components/HorizontalThermometer/CommunityLevelThermometer/CommunityLevelThermometer.tsx

```diff
@@ -3,17 +3,11 @@ import {
   ColorBlock,
   ThermometerContainer,
   Label,
-} from './RiskLevelThermometer.style';
+} from './CommunityLevelThermometer.style';
 import { Level } from 'common/level';
 import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
 
-const levels = [
-  Level.LOW,
-  Level.MEDIUM,
-  Level.HIGH,
-  Level.CRITICAL,
-  Level.SUPER_CRITICAL,
-];
+const levels = [Level.LOW, Level.MEDIUM, Level.HIGH];
```

Comment:
I think we still need the top two levels. Reducing this to three levels result in three colors in the thermometer although the map would still have five colors. This might be confusing to users since they won't know what red and dark red mean on the map.
![image](https://user-images.githubusercontent.com/46338131/159809928-395929b3-b1f9-4686-bf5b-639965ccf3fb.png)

Reply:
In the future there will only be 3 colors, this only update the thermometer

Reply:
I think ideally we should update the map and the thermometer at the same time to minimize confusion for the user, but if the map will be updated in the near future, we can move forward 😄

Reply:
This is in a feature branch - no user confusion should occur :). But potential for developer confusion is there.